tests/spec/core/biblio-spec.js

```diff
@@ -92,26 +117,148 @@ describe("W3C — Bibliographic References", () => {
     expect(ref.textContent).toMatch(/Publishers Inc\.\s/);
     ref = null;
     // Make sure the ". " is automatically added to publisher.
-    ref = doc.querySelector("#bib-TestRef2 + dd");
+    ref = docDefault.querySelector("#bib-TestRef2 + dd");
```

Comment:
Ok, here looks like you are testing a new thing (ref = ...), so it's best to split this test into a new:
```
 it("makes sure the '. ' is automatically added to publisher.", () =>{
    const ref =  docDefault.querySelector("#bib-TestRef2 + dd");
    // and so on... 
 })
```